tests/queries/0_stateless/03144_parallel_alter_add_drop_column_zookeeper_on_steroids.sh

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
+# Tags: zookeeper, no-parallel, no-fasttest
```

Comment:
This test is flaky - https://play.clickhouse.com/play?user=play#U0VMRUNUIGNoZWNrX25hbWUsIHRlc3RfbmFtZSwgcmVwb3J0X3VybApGUk9NIGNoZWNrcwpXSEVSRSB0ZXN0X25hbWUgTElLRSAnJTAzMTQ0X3BhcmFsbGVsX2FsdGVyX2FkZF9kcm9wX2NvbHVtbl96b29rZWVwZXJfb25fc3Rlcm9pZHMlJwogICAgQU5EIGNoZWNrX3N0YXJ0X3RpbWUgPj0gbm93KCkgLSBJTlRFUlZBTCAyNCBIT1VSCi8vICAgIEFORCBwdWxsX3JlcXVlc3RfbnVtYmVyID0gMAogICAgQU5EIHRlc3Rfc3RhdHVzICE9ICdTS0lQUEVEJwogICAgQU5EIHRlc3Rfc3RhdHVzIExJS0UgJ0YlJwogICAgQU5EIGNoZWNrX3N0YXR1cyAhPSAnc3VjY2VzcycKT1JERVIgQlkgY2hlY2tfbmFtZSwgdGVzdF9uYW1lLCBjaGVja19zdGFydF90aW1l